### ISSUE TITLE
File icon size too big

### ISSUE THREAD

**Issue by Blacksmoke16** [intellij-crystal/intellij-crystal#4]
The icon size for the Crystal file icon in the Project view is too big and causes issues:

![image](https://user-images.githubusercontent.com/12136995/42719527-00f572dc-86e5-11e8-8335-d87bc7243da9.png)

```
RubyMine 2018.1.3
Build #RM-181.4892.67, built on May 16, 2018
JRE: 1.8.0_152-release-1136-b38 amd64
JVM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
Linux 4.9.0-6-amd64
```

**Comment by vlad-ro** [comment]
Looks like this should be fixed with pull request https://github.com/crystal-lang-tools/intellij-crystal-lang/pull/5

**Comment by vlad-ro** [comment]
A new version of the plugin with fix for this issue and support for 2018.2 onwards has been published: https://plugins.jetbrains.com/plugin/11232-crystal-language-support-updated-for-v2018-2-onwards
It is a fork as the original plugin got dormant.
